certhandler/modules/swmodule/swmodule.go

```diff
@@ -72,8 +72,6 @@ type moduleConfig struct {
  * Vars
  ******************************************************************************/
 
-var ecsdaCurveID = elliptic.P384()
```

Comment:
This is default ecsdaCurveID - it should be as const.

Reply:
can't use const (elliptic.P384() (value of type elliptic.Curve) is not constant)
reverted to var